src/store/index.ts

```diff
@@ -1,29 +1,11 @@
 // Libraries
-import { createPinia, PiniaVuePlugin } from 'pinia'
-import Vue from 'vue'
-import Vuex from 'vuex'
-
-// Store modules
-import * as State from './state'
-
-/**
- * Configures and returns Vuex Store.
- */
-export function getVuexStore () {
-  Vue.use(Vuex)
-
-  const store = new Vuex.Store<any>({
-    state: { ...State }
-  })
-
-  return store
-}
+import { createPinia } from 'pinia'
 
 /**
  * Configures and returns Pinia Store.
  */
 export function getPiniaStore () {
-  Vue.use(PiniaVuePlugin)
+  // Vue.use(PiniaVuePlugin)
```

Comment:
This needs to be done in main.ts now?

Reply:
Just confirmed, this is a plugin for Pinia to work with Vue 2. We don't need it with Vue 3!